README.md

```diff
@@ -20,6 +20,13 @@ except: urllib.request.urlretrieve(url, filename)
 model.forward(filename)
 ```
 
+If you want to modify hyperparameters in `torchvggish.vggish_params`,
+add the following `install_requires` to your `setup.py`:
+```
+"torchvggish @ git+https://github.com/neuralaudio/torchvggish.git@setup.py#egg=torchvggish",
```

Comment:
If you approve this PR and will merge it, I will change the URL to the main repo with branch master.